src/Microsoft.Health.Dicom.Api/Configs/SwaggerConfiguration.cs

```diff
@@ -0,0 +1,19 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using Microsoft.OpenApi.Models;
+
+namespace Microsoft.Health.Dicom.Api.Configs
+{
+    public class SwaggerConfiguration
+    {
+        public Uri ServerUri { get; set; }
+
+        public string Title { get; } = "Medical Imaging Server for DICOM";
```

Comment:
Just wondering why we have the title set here and the license information in the appsettings.

Reply:
I thought this made sense as the default as it wouldn't likely change. Additionally, in looking at the underlying `OpenApiInfo`, the `title` is a required field and the `license` doesn't appear to be. 

Essentially, this should allow our service to startup without specifying anything for the `SwaggerConfiguration` and it should still spit out a valid swagger file.